packages/yamjs-plugin/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@yam-js/plugin",
   "private": true,
-  "version": "0.0.4",
+  "version": "0.0.5",
```

Comment:
Did you do this manually? If so, can you undo it?

Reply:
Ah, yeah that'll double-bump it. My bad!